extensions/cli/landsat8/src/main/java/mil/nga/giat/geowave/format/landsat8/BandFeatureIterator.java

```diff
@@ -36,6 +36,16 @@
 		PATH_ROW_FORMATTER.setMaximumIntegerDigits(3);
 		PATH_ROW_FORMATTER.setMinimumIntegerDigits(3);
 	}
+	private static final String[] SCENE_ATTRIBUTES = new String[] {
```

Comment:
same comment as AccumuloKDEReducer - we don't want copy and pasted constants in different classes, not sure what sonarqube was complaining about, but this would not be a preferable answer